e2e-tests/test/zks-apis.test.ts

```diff
@@ -57,3 +59,18 @@ describe("zks_getTokenPrice", function () {
     expect(response).to.equal("1500");
   });
 });
+
+describe("zks_getTransactionDetails", function () {
+  it.only("Should return transaction details for locally-executed transactions", async function () {
```

Comment:
Remove `.only` please
```suggestion
  it("Should return transaction details for locally-executed transactions", async function () {
```